packages/dataviews/src/item-actions.tsx

```diff
@@ -78,11 +125,14 @@ function ActionWithModal( { action, item, ActionTrigger } ) {
 	);
 }
 
-function ActionsDropdownMenuGroup( { actions, item } ) {
+function ActionsDropdownMenuGroup( {
+	actions,
+	item,
+}: ActionsDropdownMenuGroupProps ) {
 	return (
 		<DropdownMenuGroup>
 			{ actions.map( ( action ) => {
-				if ( !! action.RenderModal ) {
+				if ( 'RenderModal' in action ) {
```

Comment:
This is more readable, I like it. Is this related to any TypeScript thing I should be aware of?

Reply:
yes, basically when you do this, typescript is able to figure out the type of action that has the "RenderModal" defined. If you keep the old check, typescript will complain that RenderModal may or may not be defined.